libraries/AP_HAL_ChibiOS/hwdef/ZeroOneX6/hwdef.dat

```diff
@@ -0,0 +1,342 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the ZeroOne X6 hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 2
+
+# board ID for firmware load
+APJ_BOARD_ID 5600
+
+FLASH_RESERVE_START_KB 128
+
+# to be compatible with the px4 bootloader we need
+# to use a different RAM_MAP
+env USE_ALT_RAM_MAP 1

```

Comment:
You only need this if you are planning on putting the PX4 bootloader on the board

Reply:
Yes, We are planning on putting the PX4 bootloader on the board.

Reply:
ok